facts.txt

```diff
@@ -25,4 +25,5 @@ If you'd been around for the earliest Halloween celebrations, you might have wor
 Originally, you had to dance for your "treat."
 The Village Halloween parade in New York City is the largest Halloween parade in the United States. The parade includes 50,000 participants and draws over 2 million spectators.
 Halloween is a contraction of "All Hallows' Eve". 
-
```

Comment:
You don't need to have a blank line. readFile function checks new facts for every \n.